src/adapters/mc/asb/CMakeLists.txt

```diff
@@ -19,26 +19,15 @@ target_link_libraries(OsConfigResourceAsb
         asb)
 
 # SO binary name must match the universal class name in MOF with a "lib" prefix added
-set_target_properties(OsConfigResourceAsb PROPERTIES OUTPUT_NAME OsConfigResource)
-
-# The binary size limit for the SO binary, in bytes 
-set(MAX_FILE_SIZE 762880)
-file(WRITE "${PROJECT_BINARY_DIR}/check_filesize.cmake"
-    "file(READ filesize.txt actual_file_size)\n"
-    "if(actual_file_size GREATER ${MAX_FILE_SIZE})\n"
-        "message(FATAL_ERROR \"SO binary size exceeds the maximum allowed size of ${MAX_FILE_SIZE} bytes\")\n"
-    "endif()\n")
+set_target_properties(OsConfigResourceAsb PROPERTIES OUTPUT_NAME libOsConfigResource)
```

Comment:
just call it `OsConfigResource` as lib is prefixed automatically

Reply:
This was reverted to what was before.